Zotero/Scenes/Detail/PDF/Views/PDFAnnotationsViewController.swift

```diff
@@ -153,6 +157,21 @@ final class PDFAnnotationsViewController: UIViewController {
         }
     }
 
+    private func updateUI(state: PDFReaderState, forceReload: Bool = false, animatedDifferences: Bool = true, completion: (() -> Void)? = nil) {
+        updateQueue.async { [weak self] in
+            guard let self else { return }
+            var snapshot = NSDiffableDataSourceSnapshot<Int, PDFReaderState.AnnotationKey>()
+            snapshot.appendSections([0])
+            snapshot.appendItems(state.sortedKeys)
+            if forceReload {
+                snapshot.reloadSections([0])
```

Comment:
I'd simplify `updateUI` to previous version, which didn't have `forceReload` and keep this as "update to current state and reload updated annotations if-needed".

Reply:
Check https://github.com/zotero/zotero-ios/pull/1040/commits/19acd1e7612bb3e5dcc261eeb6f3c6f0b963d740